flexget/components/parsing/parsers/parser_guessit.py

```diff
@@ -157,6 +157,8 @@ def _quality(self, guessit_result):
         # unlike the other components, audio can be a bit iffy with multiple codecs, so we limit it to one
         if 'dts' in audio and any(hd in audio_profile for hd in ['hd', 'master audio']):
             audio = ['dtshd']
+        elif '5.1' in audio_channels and any(dd in audio for dd in ['dolby digital plus']):
```

Comment:
We don't need the 'any' here do we?
```suggestion
        elif '5.1' in audio_channels and 'dolby digital plus' in audio:
```

Reply:
To be honest, I just duplicated the following lines, which don't need an 'any' either (anymore). I amended my PR.

Reply:
Gotcha. Well, looks good to me now, thanks!